adr/0005-focus-group-process.adoc

```diff
@@ -0,0 +1,160 @@
+= Focus Groups
+
+* Status: _proposed_
+* Date: 2024-07-01 by @cescoffier
+
+== Context and Problem Statement
+
+Quarkus is a large project with many contributors.
+It's hard to keep track of all the initiatives and ensure that the community is aware of ongoing work.
+We need a way to organize work around specific topics and ensure that the community is aware of these initiatives.
+
+We also need to ensure that the work is done transparently and that the community can participate in the discussions.
+
+This would be the basis of an informal roadmap, where the community can see what is being worked on and what is coming next.
+Our previous attempts to publish and maintain a roadmap were not successful.
+We need a more lightweight approach, focusing on current work and next steps.
+
+Additionally, new contributors may find it hard to find a way to contribute to the project, as the project's size may be overwhelming.
+Working on a specific area or topic may be more appealing to new contributors and may help them get started.
+
+== Focus Groups
```

Comment:
i've looked in dictionaries and definitions and I think we should reconsider using focus group.

alternatives:

- task force
- action group
- working group 

?

Reply:
Well, none of these looks good to me.

Task force - that's not the right meaning. Force is not what we want to convey.

Working group - it's portably the less bad, but comes with a connoted process in mind. 

Action group - not sure what it means, but 'action' like 'force' is a bit too close to military wording.

At some point we mentioned focused initiatives. A bit long, but convey the right meaning.

Reply:
Working group has an advantage over the others. Using the short form (wg-) is relatively well understood. 

For example: wg-websocket-next would be relatively well understood.

Reply:
true, and fi-websocket-next for "focused initiative" sounds like financial bitcoin mining.

I'm fine with working group

Reply:
Yes, let me change to working group.

Reply:
Done.